index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <section class="first-sector">
+        <div class="header-comtainer">
+            <div class="header">
+                <div class="my-logo">My Logo</div>
+                <nav class="nav-bar">
+                    <div class="menu-buttom"><a href="#"><img src="images/humburger-btn.svg" alt="menu-buttom"></a></div>
+                </nav>
+            </div>
+        </div>
+        <div class="content">
+            <section class="main">
+            <h2 class="my-name">
+                I'm Carlos <br> Glad to see you!
+            </h2>
+            <div class="my-work">
+            <p>I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded,
+            don’t hestiate to contact me.</p>
+            </div>
+            <div class="connect">LET'S CONNECT</div>
```

Comment:
- [ ] This represents a kind of a "heading" for the social media icons, so it would be better described by an `<hx>` tag, I would suggest you use the `<h2>` one to be consistent with the heading levels of your page, this could improve a lot the experience of assistive technologies such as screen readers, ending up improving the accessibility of your page, you're going to learn this later on the program! 🤓 🛠️